lang-ca.js

```diff
@@ -924,9 +930,11 @@ SnapTranslator.dict.ca = {
     'add comment':
         'afegeix un comentari',
     'undrop':
-        'recupera bloc',
+        'desfés',
     'undo the last\nblock drop\nin this pane':
-        'recupera l\'últim bloc\nque s\'hagi llençat',
+        'desfés l\'últim moviment\nde blocs',
+    'redrop':
+        'refés',
```

Comment:
Oh I see! I stand corrected then :)

I thought you meant to keep undrop/redrop along with @brollb's undo/redo, but if there's only to be one single mechanism then @jguille2's translation makes perfect sense.